samples/TypeScript/conditionParser.mts

```diff
@@ -1,3 +1,5 @@
+const thisFile = await import.meta.url;
```

Comment:
Top-level await, nice 😄 Just not that useful...

Reply:
Yep, I initially wanted to do `await import.meta.resolve(...)`, but TS was complaining about `.resolve` not being defined so I just used url and forgot to remove await :stuck_out_tongue: 

I already pushed a new commit to remove it.